File: OpenGL_note/note_OC_01/note_OC_01/ViewController.m
//
//  ViewController.m
//  note_OC_01
//
//  Created by qiansheng on 2017/11/13.
//  Copyright © 2017年 qiansheng. All rights reserved.
//

#import "ViewController.h"
#import "OpenView.h"
@interface ViewController ()
@property (nonatomic , strong) OpenView*   myView;
@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    self.myView = (OpenView *)self.view;
    short int a = -12345;
    unsigned short  b = a;
    NSLog(@"%d",b);
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: OpenGL_note/note_01/note_01/ViewController.swift
//
//  ViewController.swift
//  note_01
//
//  Created by qiansheng on 2017/11/9.
//  Copyright © 2017年 qiansheng. All rights reserved.
//

import UIKit
import renttaxi
class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        let openView = OpneGLView(frame: self.view.bounds)
        let vc = RenttaxiController()
        
        
        
        
        
        
        //QSLog.DDLog(message: self)
        //NumberFormatterTool().changeFormatter()
        gcdasync().asfunction()
        //self.view.addSubview(openView)
        
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

